pf4j/src/main/java/org/pf4j/PluginManager.java

```diff
@@ -100,6 +100,11 @@
      */
     PluginState stopPlugin(String pluginId);
 
+    /**
+     * Unload all plugins
+     */
+    void unloadPlugins();
```

Comment:
Add implementation here and use `default` classifier on method. You can use `getResolvedPlugins` instead of `resolvedPlugins`.